ynh_systemd_action/ynh_systemd_action

```diff
@@ -74,3 +62,13 @@ ynh_systemd_action() {
         ynh_clean_check_starting
     fi
 }
+
+# Clean temporary process and file used by ynh_check_starting
+# (usually used in ynh_clean_setup scripts)
+#
+# usage: ynh_clean_check_starting
+ynh_clean_check_starting () {
```

Comment:
I don't really understand why you take this out of the helper ?

Reply:
Because of that https://github.com/YunoHost-Apps/Experimental_helpers/pull/18#issuecomment-386898015

Reply:
Ok